addons/website_event_track/static/src/js/event_track_reminder.js

```diff
@@ -82,6 +148,20 @@ publicWidget.registry.websiteEventTrackReminder = publicWidget.Widget.extend({
         }
     },
 
+    _fillIcon: function (ev) {
+        var $el = $(ev.target);
+        if ($el.attr('title') == _t('Set Favorite')){
+            $el.removeClass('fa-bell-o').addClass('fa-bell');
+        }
+    },
+
+    _emptyIcon: function (ev){
+        var $el = $(ev.target);
+        if ($el.attr('title') == _t('Set Favorite')){
+            $el.removeClass('fa-bell').addClass('fa-bell-o');
+        }
+    },
+
```

Comment:
could probably be a single function, something like this (also not sure we really need to differentiate based on the title):
```suggestion
    _onMouseEventUpdateIcon: function (ev) {
        ev.target.classList.toggle('fa-bell-o');
        ev.target.classList.toggle('fa-bell');
    },
```